scripts/test-upgrade.sh

```diff
@@ -29,3 +29,12 @@ RPCD_DIR="$BASE_DIR/rpcd"
 #        to fail to run.
 cd ${RPCD_DIR}/playbooks
 ansible -m file -a "path=/var/www/repo/openstackgit/horizon-extensions state=absent" repo_all
```

Comment:
We don't need this if we're going to destroy the repo containers, right? Let's remove this now.